rust/src/srtp_parameters.rs

```diff
@@ -13,6 +15,15 @@ pub struct SrtpParameters {
     pub key_base64: String,
 }
 
+impl SrtpParameters {
+    pub(crate) fn from_fbs(tuple: &transport::SrtpParameters) -> Self {
+        Self {
+            crypto_suite: SrtpCryptoSuite::from_str(tuple.crypto_suite.as_ref()).unwrap(),
```

Comment:
`SrtpCryptoSuite::from_str(x)` is the same as `x.parse()`, you already used it in other places and wouldn't need to import `FromStr` trait.

Reply:
done.